mantidproject/mantid
# Add missing indirect fit functions

**Description of work.**
This PR adds a set of three fit functions that were not available previously in the fit browser (`IsoRotDiff`, `DiffSphere` and `DiffRotDiscreteCircle`). These are ImmutableCompositeFunctions that were not added.

**To test:**

1. Load some data.
2. Plot data
3. Open the fit browser.
4. Add `IsoRotDiff` and try to fit,
5. Repeat with `DiffSphere` and `DiffRotDiscreteCircle`
6. Open Indirect Data analysis
7. Go to the convfit tab
8. Select `DiffSphere` and `DiffRotDiscreteCircle` and make sure they wiork

Fixes #28530

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
